WordPress/Classes/ViewRelated/Gutenberg/Collapsable Header/CollapsableHeaderViewController.swift

```diff
@@ -118,7 +126,7 @@ class CollapsableHeaderViewController: UIViewController, NoResultsViewHost {
     }
 
     private var shouldUseCompactLayout: Bool {
-        return traitCollection.verticalSizeClass == .compact
+        return alwaysShowHeaderTitles ? false : (traitCollection.verticalSizeClass == .compact)
```

Comment:
[Nitpick/Optional]: I believe this ternary can be replaced with: 
```swift
return !alwaysShowHeaderTitles && traitCollection.verticalSizeClass == .compact
```

Reply:
Oh yea that's better! Thanks!